src/tools/makeTestfile.pl

```diff
@@ -156,6 +156,9 @@ BEGIN
     alarm \$timeout;
     waitpid \$pid, 0;
     alarm 0;
+    if ($? & 127) {
```

Comment:
Core Developers: The `$?` here needs to be `\$?` (as below), since this code is inside a heredoc. We don't like the magic number and would prefer to see one the appropriate WIF macro from the Perl POSIX module.